src/map/map.hpp

```diff
@@ -507,7 +507,7 @@ enum _sp {
 	SP_WEAPON_ATK_RATE, SP_WEAPON_MATK_RATE, SP_DROP_ADDRACE, SP_DROP_ADDCLASS, SP_NO_MADO_FUEL, // 2083-2087
 	SP_IGNORE_DEF_CLASS_RATE, SP_REGEN_PERCENT_HP, SP_REGEN_PERCENT_SP, SP_SKILL_DELAY, SP_NO_WALK_DELAY, //2088-2092
 	SP_LONG_SP_GAIN_VALUE, SP_LONG_HP_GAIN_VALUE, SP_SHORT_ATK_RATE, SP_MAGIC_SUBSIZE, SP_CRIT_DEF_RATE, // 2093-2097
-	SP_MAGIC_SUBDEF_ELE // 2098
+	SP_MAGIC_SUBDEF_ELE, SP_POW, SP_STA, SP_WIS, SP_SPL, SP_CON, SP_CRT // 2098-2104
```

Comment:
share my packet extract from wireshark

SP_POW=219,
SP_STA,
SP_WIS,
SP_SPL,
SP_CON,
SP_CRT,
SP_PATK,
SP_SMATK,
SP_RES,
SP_MRES,
SP_HPLUS,
SP_CRATE,
SP_TPOINT,
SP_AP,
SP_MAXAP,
	
SP_UPOW=247,
SP_USTA,
SP_UWIS,
SP_USPL,
SP_UCON,
SP_UCRT,

Reply:
Thanks! I'll make another PR for the more detailed 4th job stuff. This was mainly for adding support to the job_db so it's ready and supported when it reaches master!